Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/tests.module.ts

```diff
@@ -14,19 +15,25 @@ import { TestQuestionsComponent } from './test-questions/test-questions.componen
 import { TestViewComponent } from './test-view/test-view.component';
 import { ClipboardModule } from 'ngx-clipboard';
 import { Test } from './tests.model';
+import { TestPreviewComponent } from './test-preview/test-preview.compponent';
 import { CreateTestHeaderComponent } from './shared/create-test-header/create-test-header.component';
 import { CreateTestFooterComponent } from './shared/create-test-footer/create-test-footer.component';
 
 import { DuplicateTestDialogComponent } from './tests-dashboard/duplicate-test-dialog.component';
-
 import { DeselectCategoryComponent } from './test-sections/deselect-category.component';
 import { IncompleteTestCreationDialogComponent } from './test-settings/incomplete-test-creation-dialog.component';
 
+import { ConductService } from '../conduct/conduct.service';
+import { ConductModule } from '../conduct/conduct.module';
+
 @NgModule({
     imports: [
         SharedModule,
         testsRouting,
-        ClipboardModule       
+        ClipboardModule,
```

Comment:
@asif-khan17 : This is a collaborative project, so you may check which developer did it and after coordination you should do the requested changes.